Fixed #36331 -- Reverted "Fixed #36055 -- Prevented overlap of object-tools buttons and page header in the admin." (#19389)
django/django: docs/releases/5.2.1.txt

* Fixed a regression in Django 5.2 where the password validation error message
  from ``MinimumLengthValidator`` was not translated when using non-English
  locales (:ticket:`36314`).

> * Fixed a regression in Django 5.2 that caused the object-tools block to be 
>   rendered twice or raised an error when using custom admin templates due to
>   changes in the block structure (:ticket:`36331`).

Reviewer: ```suggestion
* Fixed a regression in Django 5.2 that caused the ``object-tools`` block to be
  rendered twice when using custom admin templates with overridden blocks due
  to changes in the block structure (:ticket:`36331`).
```
I think "or raise an error" is not very precise and the double render example should be enough :thinking:

Author: I agree. Thank you :)
